packages/model-viewer/src/test/features/scene-graph/texture-info-spec.ts

```diff
@@ -99,10 +98,8 @@ suite('scene-graph/texture-info', () => {
                                        .pbrMetallicRoughness['baseColorTexture']
                                        .texture?.sampler!;
           expect(exported_sampler.rotation).to.be.eq(0.1, 'rotation');
-          expect(exported_sampler.offset)
-              .to.be.eql(new Vector2(0.2, 0.3), 'offset');
-          expect(exported_sampler.scale)
-              .to.be.eql(new Vector2(0.4, 0.5), 'scale');
+          expect(exported_sampler.offset).to.be.eql({u: 0.2, v: 0.3}, 'offset');
```

Comment:
I think the trouble is you're actually returning a `Vector2D` which has a toString method. I would just separate these lines to check the u and v separately.